scripts/main.js

```diff
@@ -46,48 +46,17 @@ function openOverlayWithImage(imgUrl) {
   overlayElement.classList.remove('hidden');
 }
 
-function addThumbnailClickListeners() {
+function addClickListeners(container, filter) {
 
-  // $(document.body).on('click', function (event) {
-  //   console.log('at the body');
-  // });
 
-  $(THUMBNAIL_CONTAINER_SEL).on('click', 'a', function (event) {
+  $(container).on('click', filter, function (event) {
     event.preventDefault();
     console.log('at the thumbnail container (delegated)');
     // console.log(event.target);
     console.log(event.currentTarget);
     var url = $(event.currentTarget).attr('href');
     openOverlayWithImage(url);
   });
-
-  // $(THUMBNAIL_CONTAINER_SEL).on('click', function (event) {
-  //   event.preventDefault();
-  //   console.log('at the thumbnail container');
-  //   // console.log(event.target);
-  //   // openOverlayWithImage(url);
-  // });
-
-  // $('img').on('click', function (event) {
-  //   console.log('at the img');
-  // });
-
-  // $('a').on('click', function (event) {
-  //   console.log('at the a');
-  // });
-
-
-  // var thumbnailItems = document.querySelectorAll(ALL_THUMBNAILS_SEL);
-  // thumbnailItems.forEach(function (thumb) {
-  //   var url = thumb.getAttribute('href');
-  //   thumb.addEventListener('click', function (event) {
-  //     event.preventDefault();
-  //     openOverlayWithImage(url);
-  //   })
-  //   console.log('done with addEventListener');
-  // })
-  // console.log('done with forEach');
-  // console.log('done with addThumbnailClickListeners');
```

Comment:
Would it be easier to read if we split up the old code removal into a separate commit, with an explanation of why?

Reply:
Yes.